backend/de/api/v1.py

```diff
@@ -212,6 +218,7 @@ def run_differential_expression(q: CensusCubeQuery, criteria1, criteria2) -> Tup
     - q: CensusCubeQuery object
     - criteria1: The first set of criteria for differential expression analysis.
     - criteria2: The second set of criteria for differential expression analysis.
+    - excludeOverlappingCells: A string specifying how overlapping cells should be handled.
 
     Returns:
     A tuple containing two elements:
```

Comment:
do we need to update the return? It seems to be returning 3 things now? E.g., `return [], n_overlap, 1`